include/ensmallen_bits/sgdr/cyclical_decay.hpp

```diff
@@ -131,6 +133,8 @@ class CyclicalDecay
 
   //! Locally-stored epoch.
   size_t epoch;
+  //Locally-stored min step size_t
```

Comment:
Can you fix the style here please?  `//! Locally-stored minimum step size.`  The comment doesn't make very much sense as it is.